pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+pythonpath = src
```

Comment:
This initialization file adds the `src` directory to `pytest`s `pythonpath`, meaning it can import functions, etc. from the library.